lib/plugins/aws/deploy/index.js

```diff
@@ -78,7 +78,12 @@ class AwsDeploy {
 
       // Deploy outer lifecycle
       'deploy:deploy': () => BbPromise.bind(this)
-        .then(() => this.serverless.pluginManager.spawn('aws:deploy:deploy')),
+        .then(() => {
+          if (!this.options.noDeploy) {
```

Comment:
**Aside** Just a quick, debatable comment about `if` usage / code style.

This took me quite a while to understand (since it's an inverted if).

IMO using non inverted `if`s is easier to read and understand.

```javascript
if (this.options.noDeploy) {
  return BbPromise.resolve():
}
return this.serverless.pluginManager.spawn('aws:deploy:deploy');
```